contract/src/test/java/io/neow3j/contract/NeoNameServiceTest.java

```diff
@@ -380,11 +379,11 @@ public void setRecord_typeAAAA() throws IOException {
         setUpWireMockForInvokeFunction(IS_AVAILABLE, "invokefunction_returnFalse.json");
 
         byte[] expectedScript = new ScriptBuilder().contractCall(NeoNameService.SCRIPT_HASH, SET_RECORD,
-                asList(string("client1.neo"), integer(28), string("1234:0:0:0:0:0:0:1234")))
+                asList(string("client1.neo"), integer(28), string("1234::1234")))
```

Comment:
if we don't have a test for it, yet, I would create a test case for `1234:0:0:0:0:0:0:1234` as well. To make sure.

Reply:
Actually, I didn't need to change it in the first place, since the regex is hardcoded here and in the neo-core. I just liked to change it because the `::` didn't work on the previous regex in the neo-core. ^^
So I can change it back or add another test, but it won't add better coverage or anything.

Reply:
Regex tend to be different across some languages... I don't know if dotnet's version (or regex standard) is the same as JVM-based projects. That's why I asked.

I think that adding an additional test wouldn't hurt.